cmd/webhook-manager/app/util.go

```diff
@@ -46,7 +46,7 @@ func addCaCertForWebhook(kubeClient *kubernetes.Clientset, service *router.Admis
 		var mutatingWebhookName = volcanoAdmissionPrefix + strings.ReplaceAll(service.Path, "/", "-")
 		var mutatingWebhook *v1.MutatingWebhookConfiguration
 		webhookChanged := false
-		if err := wait.Poll(time.Second, 5*time.Minute, func() (done bool, err error) {
+		if err := wait.PollUntilContextTimeout(context.Background(), time.Second, 5*time.Minute, false, func(_ context.Context) (done bool, err error) {
```

Comment:
How about set the immediate to true?

Reply:
alright, setting immediate to true more closely aligns with the behavior of wait.Poll,  i will make this change in this pr, and do the same adjustment in another pr.